keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -177,22 +177,27 @@ type PersistentVolumeClaimStatus struct {
   ...
   // The VolumeAttributesClassName string cannot be empty
   VolumeAttributesClassName string
-  ModifyVolumeStatus *PersistentVolumeClaimModifyVolumeStatus
+  ModifyVolumeStatus ModifyVolumeStatus
   ...
 }
 
+type ModifyVolumeStatus struct {
+  ActualClassName string
```

Comment:
I am wondering whether we should actually make this field an object reference instead of just a name.

I am thinking of the scenario where someone may delete and recreate the VAC with the same name but different content. But actually, if we really want to keep track of the attributes are, we may need to keep track of it in the PV object. @jsafrane @gnufied wdyt?

Reply:
Is uuid enough to distinguish instead of doing comparision of the whole VAC object?